src/goto-programs/goto_convert_class.h

```diff
@@ -97,7 +97,6 @@ class goto_convertt:public messaget
 
   void rewrite_boolean(exprt &dest);
 
-  static bool has_sideeffect(const exprt &expr);
```

Comment:
Why is this removed?

Reply:
It was never defined, so I removed the declaration as well.

Reply:
Sounds good.